src/fragment-components/a-button.tsx

```diff
@@ -42,9 +42,9 @@ export const AButtonStyleUI = ({selectedComponent, setComponent}: any) => {
 			onChange={(event: any) => setComponent({
 				...selectedComponent,
 				kind: event.selectedItem.id
-		})}/>
+			})}/>
```

Comment:
Indentation looks off for this whole return block? 🤔 Might be just github using different sizes for tabs?